File: ejercicios-py/ejercicio6.py
def factorial(num):
    if num == 0:
        return 1
    else:
        return num * factorial(num - 1)

# Calcular el factorial de un número aleatorio (ej: 5! = 120)
num = random.randint(1, 10)
resultado = factorial(num)
print(f"{num}! = {resultado}")

File: ejercicios-js/ejercicio6.js
console.log ("ejercicio 6: factorial de un numero")
function factorial(n) {
    let resultado = 1;
    for (let i = 1; i <= n; i++) {
        resultado *= i;
    }
    return resultado;
}

console.log(factorial(7)); // Salida: 5040